src/js/apps/programs/sidebar/program-sidebar_app.js

```
@@ -2,33 +2,47 @@
+        if (isNew) Radio.request('sidebar', 'stop');
       }, ({ responseData }) => {
         const errors = this.program.parseErrors(responseData);
-        this.getView().showErrors(errors);
+        this.getChildView('content').showErrors(errors);
       });
   },
+  onClose() {
+    this.stop();
+  },
   onStop() {
     if (this.program && this.program.isNew()) this.program.destroy();
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)